baggage_setter.go

```diff
@@ -22,58 +22,45 @@ package jaeger
 
 import (
 	"github.com/opentracing/opentracing-go/log"
+
+	"github.com/uber/jaeger-client-go/internal/baggage"
 )
 
 // baggageSetter is an actor that can set a baggage value on a Span given certain
 // restrictions (eg. maxValueLength).
-type baggageSetter interface {
-	setBaggage(span *Span, key, value string)
-}
-
-// defaultBaggageSetter sets the baggage key:value on the span while respecting the
-// maxValueLength and truncating the value if too long.
-type defaultBaggageSetter struct {
-	maxValueLength int
-	metrics        *Metrics
+type baggageSetter struct {
```

Comment:
I opted to just make this a struct and not an interface. Given it's private anyway, if we ever need to change it to a interface it can be easily done.